[35mDEBUG   [0m root:conftest.py:18 üë§ Criando novo usu√°rio para o teste: usuario_teste_4237
[35mDEBUG   [0m urllib3.connectionpool:connectionpool.py:1049 Starting new HTTPS connection (1): demoqa.com:443
[35mDEBUG   [0m urllib3.connectionpool:connectionpool.py:544 https://demoqa.com:443 "POST /Account/v1/User HTTP/1.1" 201 92
[35mDEBUG   [0m urllib3.connectionpool:connectionpool.py:1049 Starting new HTTPS connection (1): demoqa.com:443
[35mDEBUG   [0m urllib3.connectionpool:connectionpool.py:544 https://demoqa.com:443 "POST /Account/v1/GenerateToken HTTP/1.1" 200 288
[35mDEBUG   [0m urllib3.connectionpool:connectionpool.py:1049 Starting new HTTPS connection (1): demoqa.com:443
[35mDEBUG   [0m urllib3.connectionpool:connectionpool.py:544 https://demoqa.com:443 "POST /Account/v1/Authorized HTTP/1.1" 200 4
[35mDEBUG   [0m root:conftest.py:36 ‚úÖ Usu√°rio criado e autorizado com sucesso. ID: d282e84a-95b6-4337-917e-7d52e57e7d93, Token: eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VyTmFtZSI6InVzdWFyaW9fdGVzdGVfNDIzNyIsInBhc3N3b3JkIjoiVGVzdGVAMTIzIiwiaWF0IjoxNzM5MzcwMzAzfQ.YXLWj9Q5rM9Ec3j8GiWL7XTrq4nm1BZpXsrcginxUvg
[35mDEBUG   [0m pytest_bdd.scenario:scenario.py:150 Adding providers for fixture 'pytestbdd_stepimpl_given_que estou autenticado na API': [<FixtureDef argname='pytestbdd_stepdef_given_que estou autenticado na API' scope='function' baseid='tests/api/test_books.py'>]
[35mDEBUG   [0m root:test_books.py:15 üîê Verificando autentica√ß√£o do usu√°rio...
[35mDEBUG   [0m root:test_books.py:17 ‚úÖ Usu√°rio autenticado com sucesso!
[35mDEBUG   [0m pytest_bdd.scenario:scenario.py:150 Adding providers for fixture 'pytestbdd_stepimpl_when_envio uma requisi√ß√£o para listar os livros': [<FixtureDef argname='pytestbdd_stepdef_when_envio uma requisi√ß√£o para listar os livros' scope='function' baseid='tests/api/test_books.py'>]
[35mDEBUG   [0m root:test_books.py:22 üì§ Enviando requisi√ß√£o para listar os livros...
[35mDEBUG   [0m urllib3.connectionpool:connectionpool.py:1049 Starting new HTTPS connection (1): demoqa.com:443
[35mDEBUG   [0m urllib3.connectionpool:connectionpool.py:544 https://demoqa.com:443 "GET /BookStore/v1/Books HTTP/1.1" 200 4514
[35mDEBUG   [0m root:test_books.py:25 üì• Resposta recebida: 200
[35mDEBUG   [0m pytest_bdd.scenario:scenario.py:150 Adding providers for fixture 'pytestbdd_stepimpl_then_a API deve retornar a lista de livros dispon√≠veis': [<FixtureDef argname='pytestbdd_stepdef_then_a API deve retornar a lista de livros dispon√≠veis' scope='function' baseid='tests/api/test_books.py'>]
[35mDEBUG   [0m root:test_books.py:31 üìä Verificando resposta da API...
[35mDEBUG   [0m root:test_books.py:36 ‚úÖ Lista de livros validada com sucesso!